### DISCUSSION TITLE
Setting `ContentView.Content = someView` takes 500 ms with a non-trivial view on Windows

### DISCUSSION THREAD

**Discussion by MartyIX** [dotnet/maui#21185] [Q&A]
I have a project where we [set](https://github.com/dotnet/maui/blob/289b6c210b8c4cb34b138201f91d776cfa872b2c/src/Controls/src/Core/ContentView/ContentView.cs#L15):

```cs
contentView.Content = someVisualElement;
```

and this line alone takes 500 ms. Profiling seems to reveal that there are things like:

* ![image](https://github.com/dotnet/maui/assets/203266/77ea29d2-85ab-445a-9ec2-d2e1b90065ad)
* ![image](https://github.com/dotnet/maui/assets/203266/99fa3e9d-14c5-4fc9-817a-823c05aa2fc1)

I have no idea why `UpdateDragAndDropGestureRecognizers` takes 180 ms. That's really long. The method does not seem to do [much](https://github.com/dotnet/maui/blob/289b6c210b8c4cb34b138201f91d776cfa872b2c/src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Windows.cs#L671-L700).
 
Do you guys have any idea how to debug it further? Or is MAUI for Windows that slow right now? Or can it be my mistake?

cc @jonathanpeppers @PureWeen

**Comment by MartyIX** [comment]
cc @dotMorten If you possibly have encountered the same issue.

**Reply by jonathanpeppers** [reply]
Are you recording these `.speedscope` files in `Release` mode? Can you attach them?

**Reply by MartyIX** [reply]
I sent it via Discord.

**Reply by jonathanpeppers** [reply]
We found @MartyIX's application might be attaching drag/drop gestures on every element. We'll check again after removing this part, just to see if there are other things to fix.
